projects/js-packages/components/components/button/index.tsx

```diff
@@ -88,6 +88,6 @@ const Button = forwardRef< HTMLInputElement, ButtonProps >( ( props, ref ) => {
 		</WPButton>
 	);
 } );
-Button.displayName = 'Button';
+Button.displayName = 'button';
```

Comment:
Is there a reason for this change? It seems unrelated to the title and description of the PR.

Reply:
Restored 👍 [133e9f3](https://github.com/Automattic/jetpack/pull/41524/commits/133e9f39f67c72fd9c2e1a7e3ea2d5c8e6226b2a)